### DISCUSSION TITLE
DuckDB support

### DISCUSSION THREAD

**Discussion by carlopi** [pyscript/pyscript#2132] [General]
Hi!
Recently DuckDB support has landed in [Pyodide](https://github.com/pyodide/pyodide/pull/4684), and I was wondering what are the steps / timeline to make it so that it will be available also in pyscript / anaconda.

Main questions are like:
- which Pyodide version needs to be supported? (I looked at the examples at https://pyscript.com/@examples, and they seem to use Pyodide 0.24.1, what's the timeline for update? DuckDb is part of the packages starting with 0.26.2)
- how to be available as default installable packages? (also from the examples, I think you are using the default package.json list provided by the pyodide CDN, but then I am not sure how it's possible to be added to that list for past releases)

Thanks a lot!

**Comment by WebReflection** [comment]
examples are not always up to latest PyScript, unless needed, we ship 0.26.1 though and we were not aware of the Pyodide release (shame on us).

packages are those provided by pyodide but if you have a special place for your package you can always add it to the `config.toml`, **example**

```toml
packages = [
  "https://cdn.holoviz.org/panel/wheels/bokeh-3.5.0-py3-none-any.whl",
  "https://cdn.holoviz.org/panel/1.5.0-b.2/dist/wheels/panel-1.5.0b2-py3-none-any.whl"
]
```

This works well for *panel* and I don't see why it wouldn't for your project ... however, we don't have any way to infer packages thta are not already available via the interpreter, so if that's not part of pyodide default distribution, I am afraid you need to spread the word a bit around on how people can get your package in PyScript too.

I hope this answer makes sense and thank you for the interest in our project, feel free to ask anything else/more if needed.

**Reply by WebReflection** [reply]
Once [this gets merged](https://github.com/pyscript/pyscript/pull/2133) and we release we'll be on 0.26.2 too.
